### DISCUSSION TITLE
next/image serves image with low resolution per Lighthouse-Best Practices

### DISCUSSION THREAD

**Discussion by BernardA** [vercel/next.js#18842] [Help] [answered]
With the new `Image` component, Lighthouse is not happy with the image sizes provided as you can see below:

![Screenshot 2020-11-05 at 14 58 18](https://user-images.githubusercontent.com/9449665/98250154-6621ea00-1f77-11eb-8c55-5d6e76da8ff6.png)

After reading quite a lot about it I am still unable to satisfy Lighthouse. Take a look at [this issue](https://github.com/GoogleChrome/lighthouse/issues/10802), for instance. 

I thought that that could be addressed by the `sizes` property of the `Image` component, but that did not work.

        <Image
                            src="/images/main-logo.png"
                            alt="quiamo logo"
                            className={classes.branding}
                            width="180"
                            height="60"
                            loading="eager"
                            priority
                            sizes="only screen and (min-resolution: 3dppx) 473px"
            />

This would output like below, ie, adding a `sizes` property with a media-query, but no effect on Lighthouse.
I was expecting it to add a `srcset` with the informed size.

Actually I was expecting that `next/image` would handle that automatically by providing different sizes/resolution for the most common devices. 

    `<img alt="quiamo logo" src="/_next/image?url=%2Fimages%2Fmain-logo.png&amp;w=320&amp;q=75" srcset="/_next/image?url=%2Fimages%2Fmain-logo.png&amp;w=320&amp;q=75 320w" class="makeStyles-branding-7" sizes="only screen and (min-resolution: 3dppx) 473px" style="visibility:visible;height:100%;left:0;position:absolute;top:0;width:100%">`

**Comment by Timer** [comment, marked as answer]
It looks like you're on Next.js 10.0.0, can you please upgrade to 10.0.1+ and remove your `sizes` prop?

**Reply by BernardA** [reply]
Quite right. Thanks.

I upgraded to 10.0.1 and now both Lighthouse and I are happy.

Well, not quite. That problem went away but I got another one now. 

As you can see, it is now complaining that the dimensions are not set, when they certainly are as per  my code above.

![Screenshot 2020-11-05 at 17 04 52](https://user-images.githubusercontent.com/9449665/98265167-0f251080-1f89-11eb-831d-1c17961acefc.png)

Should I open a separate issue for that one?

**Reply by BernardA** [reply]
Hang on. Not sure which image it is referring to. Let me check.

**Reply by BernardA** [reply]
It is actually referring to the same `Image` component. See markup below.

So, Lighthouse is complaining that that additional `<img>` tag does not have the required dimensions. Not sure of the implications of that.

![Screenshot 2020-11-05 at 17 24 12](https://user-images.githubusercontent.com/9449665/98268066-5d87de80-1f8c-11eb-9fef-c00f0add0c93.png)

**Reply by Timer** [reply]
Seems like this might be a bug in lighthouse. There's a width and height defined on the SVG.

**Reply by BernardA** [reply]
ok, I will report this on Lighthouse then, thanks.

**Reply by Devang-mic** [reply]
I am facing same problem on docker image node:16-alpine.
I have added imageSizes in next config as well. 
Any solution ?

Here is my code: 
`<Image
                                src="/static/images/image.png"
                                alt="test"
                                width={145}
                                height={71}
                                priority={true}
                            />`
